src/encord_active/db/models.py

```diff
@@ -153,14 +153,17 @@ class ProjectDataAnalytics(SQLModel, table=True):
     metric_red: Optional[float] = MetricFieldTypeNormal
     metric_green: Optional[float] = MetricFieldTypeNormal
     metric_blue: Optional[float] = MetricFieldTypeNormal
+    metric_custom0: Optional[float] = MetricFieldTypeNormal
     # Random
     metric_random: Optional[float] = MetricFieldTypeNormal
     # Image Only
     metric_object_count: Optional[int] = MetricFieldTypePositiveInteger
     metric_object_density: Optional[float] = MetricFieldTypeNormal
     metric_image_difficulty: Optional[float]  # FIXME: is the output of this always an integer??
     metric_image_singularity: Optional[float] = MetricFieldTypeNormal
-
+    metric_video_red_density: Optional[float] = MetricFieldTypeNormal
```

Comment:
Could you add a comment to mark this as its own section of Data metrics.
# Whole video metrics or similar.